drivers/scsi/iscsi_tcp.c

```diff
@@ -63,8 +63,9 @@ MODULE_PARM_DESC(debug_iscsi_tcp, "Turn on debugging for iscsi_tcp module "
 		 "Set to 1 to turn on, and zero to turn off. Default is off.");
 
 #ifdef CONFIG_TRUENAS
-#define HA_PRIVATE_CHANNEL_CONTROLLER_A_IP "169.254.10.1"
-#define HA_PRIVATE_CHANNEL_CONTROLLER_B_IP "169.254.10.2"
+static char *iscsi_genhd_hidden_ips;
+module_param_named(genhd_hidden_ips, iscsi_genhd_hidden_ips, charp, 0644);
+MODULE_PARM_DESC(recv_from_iscsi_q, "Comma separated list of IP addresses from which logged in targets should be hidden.");
```

Comment:
Is there reason to not wrap this line?  I see another long example above, but the rest of the file seems to be wrapped to 80 columns.

Otherwise looks good to me.